python/setup.py

```diff
@@ -42,9 +42,14 @@
 DEPENDENCIES = [
     "numpy",
     "pandas",
+    "pydantic",
+    "spacy",
+    "ipywidets",
+    "transformers",
```

Comment:
can you take a look at this file:
https://github.com/interpretml/interpret-text-contrib/blob/master/tools/generate_conda_files.py
they seem to pin transformers to a specific version, similarly for some other dependencies, not sure if that is needed
the requirements.txt isn't actually used in the default readme setup or in the builds which is kind of weird

Reply:
I noticed that too. I think we took the nlp convention, I will be deleting that generate file in this PR.

Thanks for mentioning the pin, I will take a look to see if that bug is fixed in transformers/our use of that code